config/dashboard.php

```diff
@@ -24,5 +24,5 @@
         'login' => false,
         'logout' => false,
     ],
-    'auth_activity_causer' => 'users',
+    'auth_activity_causer' => 'users'
```

Comment:
Is our formatter config removing trailing commas? If you're not against them @haringsrob I would prefer to keep them. Diffs are more readable and it is less typing when adding a new line.

Reply:
Will add a standard.

Reply:
Not super relyable but I added https://github.com/squizlabs/PHP_CodeSniffer/issues/582